src/appkit.cs

```diff
@@ -6094,6 +6094,7 @@ interface NSFontCollection : NSSecureCoding, NSMutableCopying {
 
 	[Lion]
 	[BaseType (typeof (NSFontCollection))]
+	[DisableDefaultCtor]
```

Comment:
That's not a new API so it's a breaking change. You need to add an empty stub for the .ctor under a `!XAMCORE_4_0` to maintain compatibility.